soc/xtensa/intel_adsp/cavs_v25/linker.ld

```diff
@@ -476,22 +490,25 @@ SECTIONS
     *(.gna_model)
     _data_end = ABSOLUTE(.);
     . = ALIGN(4096);
-  } >ram :ram_phdr
-  .lit4 : ALIGN(4)
+  } >ucram :ucram_phdr
+
+  .lit4 SEGSTART_CACHED : ALIGN(4)
   {
     _lit4_start = ABSOLUTE(.);
     *(*.lit4)
     *(.lit4.*)
     *(.gnu.linkonce.lit4.*)
     _lit4_end = ABSOLUTE(.);
   } >ram :ram_phdr
-  .cached :
-  {
-    *(.cached .cached.*)
-  } >ram :ram_phdr
+
```

Comment:
cAVS 1.8 has `. = SEGSTART_UNCACHED;` here - switching over to the uncached alias

Reply:
Thanks. Does not seem to help. :(

Reply:
So it works without this too? Could you check e.g. something like (adjust for your tools and paths)
```
xtensa-cnl-elf-objdump -t -jsw_isr_table build/zephyr/zephyr.elf
```
are those now in `0xbe` or in `0x9e`?

Reply:
@lyakh 
```
9e046000 l    d  sw_isr_table   00000000 sw_isr_table
9e046000 g     O sw_isr_table   000004a8 _sw_isr_table
```

Reply:
Thanks. I'd still add those two things at least for uniformity, but so far we have no evidence that that matters :-)